Cartfile.resolved

```diff
@@ -1,16 +1,15 @@
 binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon-ios.json" "4.0.0"
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps/mapbox-ios-sdk-dynamic.json" "6.1.0"
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" "18.0.3"
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps/mapbox-ios-sdk-dynamic.json" "6.2.0"
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" "18.0.4"
 binary "https://www.mapbox.com/ios-sdk/MapboxAccounts.json" "2.3.0"
-github "CedarBDD/Cedar" "v1.0"
```

Comment:
I removed this line when trying to see whether I could get past the `lipo` issue by temporarily dropping Cedar, but I forgot to put it back.